htdocs/adherents/class/adherent.class.php

```diff
@@ -519,9 +526,10 @@ public function create($user, $notrigger = 0)
 
 		// Insert member
 		$sql = "INSERT INTO ".MAIN_DB_PREFIX."adherent";
-		$sql .= " (datec,login,fk_user_author,fk_user_mod,fk_user_valid,morphy,fk_adherent_type,entity,import_key)";
+		$sql .= " (ref, datec,login,fk_user_author,fk_user_mod,fk_user_valid,morphy,fk_adherent_type,entity,import_key)";
 		$sql .= " VALUES (";
-		$sql .= " '".$this->db->idate($this->datec)."'";
+		$sql .= " '(PROV)'";
```

Comment:
The ref is set to (PROV) here but where is it set to real value into transaction ? I did not find code for this.